### ISSUE TITLE
Any alert() function call error

### ISSUE THREAD

**Issue by sipy23** [timelyart/Kairos#22]
**Describe the bug**
I believe there is an issue with the 'Any alert() function call' alerts, as those do not have the 'Options' tab menu so the script stops

**Error**
```
2021-07-06 00:57:05.569 ERROR tv - create_alert: Unable to find button called 'Only Once'.
Please, check if the value provided in your yaml (still) exists on TV.
Traceback (most recent call last):
  File "C:\Kairos-2.43\tv\tv.py", line 2028, in create_alert
    wait_and_click(alert_dialog, css_selectors['checkbox_dlg_create_alert_frequency'].format(button_name))
  File "C:\Kairos-2.43\tv\tv.py", line 386, in wait_and_click
    return WebDriverWait(browser, delay).until(
  File "C:\Users\User\AppData\Local\Programs\Python\Python39-32\lib\site-packages\selenium-3.141.0-py3.9.egg\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
```

**Screenshots**
![image](https://user-images.githubusercontent.com/4532330/124591928-8e931880-de65-11eb-9640-bb2fce77e10d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 91

**Comment by timelyart** [comment]
Thank you for your report. I had no idea you could add an alert without setting any options. This must have been recently added.
Please update to release [v2.44](https://github.com/timelyart/Kairos/releases/tag/v2.44) and remove the key `options` from your YAML. 

Please, let me know if this worked for you.

**Comment by sipy23** [comment]
Hello, thanks for the prompt response!

I am not sure if tradingview did a new small update, but I can't seem to get pass the login part now (on both v2.44 or v2.43), getting this error

`2021-07-06 16:29:50.575 INFO tv - run: Running on a windows operating system
2021-07-06 16:29:51.782 INFO tv - check_driver: browser version: 91.0.4472.124
2021-07-06 16:29:51.784 INFO tv - check_driver: driver version: 91.0.4472.101
2021-07-06 16:30:10.029 ERROR tv - login: Message: element click intercepted: Element <button class="tv-header-search__input-button" type="submit">...</button> is not clickable at point (360, 30). Other element would receive the click: <div class="tv-dialog__modal-container">...</div>
  (Session info: chrome=91.0.4472.124)`

**Comment by timelyart** [comment]
Are you using the same browser while running Kairos? 

If so, you should set `user_data_directory` and set `share_user_data = true`. Otherwise, your mouse clicks and movement will effect Kairos. 

If you do not use the browser at the same time while running Kairos then, please, make a separate issue and include the screenshot that Kairos creates for this type of error.

In any case, it is unrelated to this issue.

**Comment by sipy23** [comment]
Hey there, the issue with the 'Any alert() function call' persists, I believe it may be because of the `message:` YAML key, as the message box is also missing on these type of alerts, perhaps we can make that optional aswell.

**ErrorLog**
[xtelegram.yaml.log](https://github.com/timelyart/Kairos/files/6771402/xtelegram.yaml.log)

**Comment by timelyart** [comment]
Can you also add your YAML file please? Make sure there is no sensitive data in the file.

I suspect you will need to put the value(s) for `conditions` between (double) quotes.

**Comment by sipy23** [comment]
I forgot to mention, the script correctly identified the indicator, the alert's name and adjusted the alert's name correctly, then it stopped

Here is the YAML file
[yaml file.txt](https://github.com/timelyart/Kairos/files/6771691/yaml.file.txt)

**Comment by timelyart** [comment]
Just out of curiosity what is the purpose of these kinds of alerts?

**Comment by timelyart** [comment]
Please update to release [v2.46](https://github.com/timelyart/Kairos/releases/tag/v2.46) and remove the key `message` from your YAML.

Please, let me know if this worked for you.

**Comment by sipy23** [comment]
Seems to be working now. Thanks!

The new alert() function in pine allows you to setup 1 alert based on multiple events, each with different alert messages. These alert messages can contain any dynamic information from the script. (for example we could create a JSON formatted alert message with any data from the script)
